src/moin/config/wikiconfig.py

```diff
@@ -18,10 +18,10 @@
     registration_only_by_superuser = True
 to:
     registration_only_by_superuser = False
-and change several lines containing:
-    dict(before='YOUR-SUPER-EDITOR:read,write,create,destroy,admin',
+and change four lines containing:
```

Comment:
Do you really want to count the lines that need changing every time?

Reply:
Just trying to make it simple and explicit for desktop wiki users. If the number changes, I will put it back to "several".